R/ari_spin.R

```diff
@@ -102,51 +103,59 @@ ari_spin <- function(images, paragraphs,
       "likely needs to be replaced or slide removed!"
     ))
   }
-
   stopifnot(
     length(paragraphs) > 0,
     identical(length(images), length(paragraphs)),
     all(file.exists(images)),
     dir.exists(output_dir)
   )
+  # End of Argument checks
 
-  wavs <- vector(mode = "list", length = length(paragraphs))
+  # Setup objects to populate in for-loop with tts()
+  wave_objects <- vector(mode = "list", length = length(paragraphs))
   par_along <- seq_along(paragraphs)
   ideal_duration <- rep(NA, length(paragraphs))
 
   pb <- progress_bar$new(
     format = "Fetching Narration [:bar] :percent",
     total = length(par_along)
   )
-
+  # Iterate through arguments used in tts()
   for (i in par_along) {
     args <- tts_args
     args$text <- paragraphs[i]
     args$voice <- voice
     args$service <- service
     args$bind_audio <- TRUE
+    # coqui+ari doesn't work with mp3
+    if (service == "coqui") {
+      args$output_format <- "wav"
+    }
+
     wav <- do.call(text2speech::tts, args = args)
     wav <- reduce(wav$wav, bind)
     wav <- pad_wav(wav, duration = duration[i])
     ideal_duration[i] <- length(wav@left) / wav@samp.rate
-    wavs[[i]] <- wav
+    wave_objects[[i]] <- wav
```

Comment:
Why is this different?

Reply:
I changed the variable name to be something more descriptive